microsetta_interface/server.py

```diff
@@ -77,6 +79,42 @@ def run(app):
     )
 
 
+@babel.localeselector
+def get_locale():
```

Comment:
This needs to either move below the definition of `babel` on line 119, or lines 118-119 need to be moved prior to the use of `babel.localeselector`

Reply:
Oh wow, that syntax is super weird.  I thought that babel wanted the module, but I guess you're right